src/analysis/retail/monk/windwalker/modules/talents/XuensBattlegear.tsx

```diff
@@ -6,7 +6,7 @@ import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
 import Events, { DamageEvent } from 'parser/core/Events';
 import { ThresholdStyle, When } from 'parser/core/ParseResults';
 import AbilityTracker from 'parser/shared/modules/AbilityTracker';
-import SpellUsable from 'parser/shared/modules/SpellUsable';
+import SpellUsable from 'analysis/retail/monk/windwalker/modules/core/SpellUsable';
```

Comment:
I know you only imported this into this module but while you're here you should update this statistic to be in the TALENTS category since its no longer a SL Leggo
line 105:
`category={STATISTIC_CATEGORY.ITEMS}` => `category={STATISTIC_CATEGORY.TALENTS}`